### DISCUSSION TITLE
Can you start to code at 38 Years old ?

### DISCUSSION THREAD

**Discussion by Pythonbutnotsnake** [codewars/codewars.com#3038] [General]
I have none to very basic Coding background . Lets say a total beginner but i have started learning Python recently and i was wondering if its a good move .. i want to build a remote keylogger crypter and that sorta stuff perhaps apps too .
Am i too old to start ?

**Comment by hobovsky** [comment]
Hello,

Please mind that Github Discussions are an area meant for discussions related directly to the Codewars platform and website, its functions, processes, and technical issues. Github is not a general place where community of Codewars users hangs out, and it can be difficult for you to get an answer to your question here.

I would strongly recommend visiting Codewars Discord server, and asking your question in one of its channels: create your own thread in  `#beginners`, or in `#posts` section. Traffic there is much higher, more users visits Discord than Github, and you might get better answers there.

Cheers!

**Comment by Mamangbili** [comment]
i've read 81 years old grandma start to code in Japan. does that sound too old for you?

**Comment by hobovsky** [comment]
I am closing the discussion as off-topic. If the OP wants to continue, they can recreate it on Codewars Discord in `#posts` section.
